pubsub/sqs_pub_test.go

```diff
@@ -157,5 +202,33 @@ var _ = Describe("SQS Publisher", func() {
 				Fail("timed out waiting for Publisher to exit")
 			}
 		})
+		It("will ignore OK outcomes if configured to", func() {
+			ack := protos.EventResponse{
+				EventId: "dead-beef",
+				Outcome: &protos.EventOutcome{
+					Code: protos.EventOutcome_Ok,
+				},
+			}
+			testPublisher = pubsub.NewSqsPublisher(notificationsCh, &sqsUrl, true)
+			done := make(chan interface{})
+			go func() {
+				defer close(done)
+				go testPublisher.Publish(getQueueName(notificationsQueue), getQueueName(acksQueue))
+			}()
+
+			notificationsCh <- ack
+			time.Sleep(channelWait)
+			res := getSqsMessage(getQueueName(acksQueue))
```

Comment:
out of curiousity, you may add a `getSqsMessage` for the `notificationsQueue` and confirm whether it gets the ok message (which I think it does with current code); you need to check them both anyway and confirm we drop those